### DISCUSSION TITLE
Can someone please update the builds?

### DISCUSSION THREAD

**Discussion by vdruts** [nomacs/nomacs#903] [General]
It's been over 2 years. Can someone please post a new build for Windows + OSX? Thanks so much!

**Comment by greg4b** [comment]
"Artifacts" can be found here https://ci.appveyor.com/project/diemmarkus/nomacs/builds/45270115/artifacts

**Reply by sunnz** [reply]
> Artifacts older than 1 month are automatically deleted. For more details see [Artifacts retention policy](https://www.appveyor.com/docs/packaging-artifacts/#artifacts-retention-policy).

**Comment by scrubbbbs** [comment]
New link for recent Windows builds https://ci.appveyor.com/project/novomesk/nomacs

**Reply by e-t-l** [reply]
Is this a stable build, though? There are more up-to-date builds in the github Releases page, but they are all Beta releases. v3.16 is the last "stable" release, and also the only one installable via WinGet...

**Reply by scrubbbbs** [reply]
We aren't maintaining a separate "stable" and "beta" version right now. It is a resources problem. The beta releases are the best we have, and appveyor has the "alpha" or nightly builds for those looking for a particular bugfix.

We do not have a reliable group of testers to sign off on a stable version, nor the bandwidth (desire?) to maintain separate stable and development/beta versions. Personally I would not want to maintain both versions.

**Reply by sunnz** [reply]
where can you get the builds for macos? (any builds)

**Reply by scrubbbbs** [reply]
> where can you get the builds for macos? (any builds)

Besides github releases (maybe there's a really old build there?) there aren't any. We fixed compilation on homebrew recently so that's an option (see Readme on main github page). 

I might be able to provide an x86 build, I'm looking into it now. If I can get it sorted it I will add it to the recent betas.

**Reply by scrubbbbs** [reply]
macOS intel build for 3.21.1 added today

**Comment by kekkc** [comment]
Any chance for new builds (or probably plans to automate the build process in the future)?

**Comment by scrubbbbs** [comment]
We just last week added 3.22.0 RC1 builds with Linux AppImage and macOS builds. We would appreciate if you give it a try and let us know how it works for you.

This is an important build to test, as the 3.22.0 general release will finally replace 3.16 as the default download on github, and may become a longer-term stable version provided we can get our CI to cooperate.
